api/scpca_portal/test/test_metadata_file.py

```diff
@@ -68,6 +80,7 @@ def test_write_metadata_dicts_read_write_values_match(self):
                 "country": "USA",
                 "language": "English",
                 "capital": "Washington DC",
+                "holidays": ("New Year's Day;" "Thanksgiving Day;" "Christmas Day"),
```

Comment:
```suggestion
                "holidays": "New Year's Day;Thanksgiving Day;Christmas Day",
```

The formatter usually doesn't handle this well so sometimes its better to manually update these.